include/ceed/ceed.h

```diff
@@ -199,11 +199,11 @@ CEED_EXTERN const char *const CeedCopyModes[];
 /// Denotes type of vector norm to be computed
 /// @ingroup CeedVector
 typedef enum {
-  /// L_1 norm: sum_i |x_i|
+  /// \f$L_1\f$ norm: \f$\sum_i \vert x_i\vert\f$
   CEED_NORM_1,
-  /// L_2 norm: sqrt(sum_i |x_i|^2)
+  /// \f$L_2\f$ norm: \f$\sqrt{\sum_i \vert x_i \vert^2}\f$
   CEED_NORM_2,
-  /// L_Infinity norm: max_i |x_i|
+  /// \f$L_\infty\f$ norm: \f$\max_i \vert x_i \vert\f$
```

Comment:
These are discrete 1-norm, 2-norm, $\infty$ norms. The [usual notation](https://en.wikipedia.org/wiki/Lp_space) is $L^p$ rather than $L_p$, though it's sometimes useful to distinguish the continuous norms (integrals) from these discrete norms.

Reply:
Yeah, not sure how to distinguish the continuous from the discrete form other than writing out the definition there. I don't think there's a common notation to distinguish the difference (I thought it was $L^p$ vs $\ell^p$, but according to wikipedia that's not necessarily true).

Reply:
Just saying "1-norm" or writing $\lVert x \rVert_1$ is common. The $\ell^1$ and $L^1$ and notation is usually for countably and uncountably infinite dimensional $x$.